index.html

```diff
@@ -113,9 +113,9 @@ <h2>Registration Entry Requirements</h2>
           without cost (other than reasonable shipping and handling if not available by online means).</li>
         <li>Each entry MUST comply with all requirements outlined in the <a def-id="byte-stream-formats-section"></a>
           of the <a def-id="mse-spec"></a> specification [[!MEDIA-SOURCE]].</li>
-        <li>Candidate entries MUST be announced on <a href="mailto:public-html-media@w3.org">public-html-media@w3.org</a> (<a href="mailto:public-html-media-request@w3.org">subscribe</a>,
-          <a href="http://lists.w3.org/Archives/Public/public-html-media/">archives</a>) so they can be discussed and evaluated for compliance before being added to the
-          registry.</li>
+        <li>Candidate entries MUST be announced on <a href="mailto:public-media-wg@w3.org">public-media-wg@w3.org</a> (<a href="mailto:public-media-wg@w3.org">subscribe</a>,
```

Comment:
Is that update needed?

The `public-html-media` mailing-list may continue to be used to record candidate entries. It seems important to provide a stable mechanism over time than to adjust the mailing-list to match that of the working group that currently maintains the spec.

In any case, given that the change would update a normative assertion in the spec, I think it should first be brougt to and validated by the Media WG.

Reply:
If `public-html-media` remains a viable forum for these announcements, then keeping it unchanged and stable would be better IMHO, too. I'll update this PR to leave the announcement mailing list AS-IS.

Reply:
https://github.com/w3c/mse-byte-stream-format-registry/pull/2/commits/50d774c50796706ce8c12d72b5e753a9f13ed7a4 (in this PR) switches that paragraph back to what it was before this PR, so it will remain `public-html-media` for now at least.